File: AMS_基础信息组件/code/client/source/YssInformation.Support/Bi/AccountTree/Service/IAccountTreeADataService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using FAST.Common.Service.DataService.Base;
using FAST.Common.Service.DataService;
using FAST.Common.Service.Pojo.Base;
using FAST.Common.Service.Services.Base;
using FAST.Common.Service.Attributes;

namespace YssInformation.Support.Bi.AccountTree.Service
{
    /// <summary>
    /// 账户树形结构A区DataService
    /// </summary>
    [ServiceAttribute(ServiceId = YssInformation.Support.common.ServiceIdConstant.OSGI_BASEBUSINESS, ServiceController = "com.yss.ams.base.information.support.bi.accountTreeA.controller.IAccountTreeADataController", ServiceUrl = "")]
    [NewServiceAttribute(ServiceType.NEW_SVC)]
    public interface IAccountTreeADataService : IControlDataService, IKeyConvertDataService
    {
    }
}

File: AMS_基础信息组件/code/server/source/com.yss.ams.base.information.support/src/main/java/com/yss/ams/base/information/support/bi/accountTreeA/service/IAccountTreeADataService.java
package com.yss.ams.base.information.support.bi.accountTreeA.service;

import com.yss.framework.api.dataservice.IControlDataService;
import com.yss.framework.api.dataservice.IDataService;
import com.yss.framework.api.dataservice.IKeyConvertDataService;
import com.yss.framework.api.restful.annotations.RestfulSupported;

@RestfulSupported
public interface IAccountTreeADataService extends IDataService,
IControlDataService, IKeyConvertDataService {

}
